apps/docs/pages/guides/getting-started/tutorials/with-nextjs.mdx

```diff
@@ -103,6 +103,7 @@ npm install @supabase/auth-helpers-react @supabase/auth-helpers-nextjs
 Wrap your `pages/_app.js` component with the `SessionContextProvider` component:
 
 ```jsx title=pages/_app.js
+import { useState } from 'react'
```

Comment:
Hi @AndreJones216,

Thanks for this – nice catch!

Sorry to be pedantic, but would you mind moving this import under the `SessionContextProvider` import? Then, it'll look nicer in the guide. Cheers!

Reply:
Sorry to be a noob, but I am not sure where this is located. This is my first time doing this.

Reply:
No worries. I mean to have the imports look like this:

```typescript
import '../styles/globals.css'
import { createBrowserSupabaseClient } from '@supabase/auth-helpers-nextjs'
import { SessionContextProvider } from '@supabase/auth-helpers-react'
import { useState } from 'react'
```